### ISSUE TITLE
Data preprocessing and conda config file

### ISSUE THREAD

**Issue by rmwu** [mims-harvard/SubGNN#5]
Hi! Thank you for all your documentation on this project. :) It's been very helpful so far! I'm trying to preprocess a custom dataset using the scripts provided, and I'm running into issues with dependencies.

I created the conda environment with the yml provided, but line 201 "torch==1.0.0" seemed to override line 68 "pytorch=1.4.0=py3.7_cuda10.1.243_cudnn7.6.3_0" so my installation was for 1.0.0. To obtain the correct version, I manually installed version 1.4.0, but that didn't seem to include the right version of torch-geometric, as GraphSAINTRandomWalkSampler appears in a later version. Would it be possible to update the environment yml with a pruned version of the requirements, or is there another sampler you would recommend?

**Comment by EmilyAlsentzer** [comment]
Hi @rmwu, you're right that the GraphSaint sampler requires a newer version of pytorch geometric. We used two different environments for `GIN` vs `GCN-GraphSaint` embeddings because pytorch geometric was updated mid-project, but I just updated everything to the new version of Pytorch Geometric. On the modeling side, the only difference is refactoring of the functions that were used in the `propagate` function in `subgraph_mpn.py`. There's a new `SubGNN.yml` file that reflects the updated environment with the new version of pytorch geometric.
